apps/test/unit/lib/levelbuilder/lesson-editor/activitiesEditorReduxTest.js

```diff
@@ -310,6 +321,18 @@ describe('activitiesEditorRedux reducer tests', () => {
         assert.deepEqual(expectedState, state.activities);
       });
 
+      it('remove last activity section in activity', () => {
+        let state = reducer(initialState, removeActivitySection(1, 1));
+        assert.deepEqual(1, state.activities[0].activitySections.length);
+
+        state = reducer(state, removeActivitySection(1, 1));
+
+        let expectedState = _.cloneDeep(initialActivities);
```

Comment:
Is this ever used in this test?

Reply:
Good catch! I had planned to check the state against expected state and it looks like I forgot.